scipy/special/_generate_pyx.py

```diff
@@ -1227,9 +1228,9 @@ def get_declaration(ufunc, c_name, c_proto, cy_proto, header,
         # redeclare the function, so that the assumed
         # signature is checked at compile time
         new_name = "%s \"%s\"" % (ufunc.cython_func_name(c_name), c_name)
-        defs.append("cdef extern from \"%s\":" % proto_h_filename)
+        defs.append(f'cdef extern from r"{proto_h_filename}":')
         defs.append("    cdef %s" % (cy_proto.replace('(*)', new_name)))
-        defs_h.append("#include \"%s\"" % header)
+        defs_h.append(f'#include "{header}"')
```

Comment:
Let me comment on the changes here, for future visibility. This is one change that's not ideal, and gh-14890 is tracking what should be done about this script.